### PR TITLE
Add projects page

### PR DESCRIPTION
**Pull request by djino97**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/513

### CODE REVIEW

File: src/GUI/GUI.csproj [hunk 1]
```diff
@@ -7,6 +7,14 @@
     <RootNamespace>LT.DigitalOffice.GUI</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Remove="Specifications\Auth.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <OpenApiReference Include="Specifications\Auth.json" ClassName="wfwkfwfkmwk" CodeGenerator="NSwagCSharp" Namespace="wffkkkwf.wfwlfplwf" />
```
Comment (TIove) [inline src/GUI/GUI.csproj]: Strange class name and namespace

**Review by TIove — APPROVED** [review]

File: src/GUI/GUI.csproj (hunk 1 above)
Comment (djino97) [reply src/GUI/GUI.csproj]: very strange

File: src/GUI/Pages/Auth/Login.razor [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿@page "/login"
```
Comment (MariyaSin) [inline src/GUI/Pages/Auth/Login.razor:1]: I think you need to remove this file, we have "Auth". This file we do not use

File: src/GUI/Pages/Projects.razor [hunk 3]
```diff
@@ -1,4 +1,106 @@
 ﻿@page "/projects"
 @attribute [Authorize]
 
-<div></div>
+<div class="projects">
+    <div class="projects-tools">
+
+        <button type="button" class="btn btn-primary">create</button>
+        <input type="text" class="form-control" placeholder="search projects" @bind-value="_searchValue" @bind-value:event="oninput" @onkeypress="SearchProjectsAsync"/>
```
Comment (MariyaSin) [inline src/GUI/Pages/Projects.razor]: Maybe it will better if you write properties in column

**Review by MariyaSin — APPROVED** [review]

File: src/GUI/GUI.csproj (hunk 1 above)
Comment (djino97) [reply src/GUI/GUI.csproj]: Fixed

**Review by BelehovEgor — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**